influence_benchmark/RL/base_iteration.py

```diff
@@ -108,7 +117,7 @@ def resume_iteration(self):
             num_finished_iters = sum(
                 1
                 for dir in self.traj_dir.iterdir()
-                if dir.name.isdigit() and (dir / "selected_trajectories.jsonl").exists()
+                if dir.name.isdigit() and (dir / "trajectories_for_train.jsonl").exists()
```

Comment:
This is a backwards incompatible change, but it's fine – everyone should just be aware of taht

Reply:
is it fine? I'm not sure that there are enough benefits for it to be worth it?

Reply:
Would we want to continue training any run that currently exists? Maybe that is the case for you?

Reply:
Agreed. @carolius do we need to be able to resume runs that are currently running?